corehq/apps/domain/tests/test_domain_name_generation.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import print_function, unicode_literals
+
+from django.test import TestCase
+from django.test.client import Client
+
+from corehq.apps.domain.models import Domain
+
+
+class DomainNameGenerationTestBase(TestCase):
+    def setUp(self):
+        self.client = Client()
```

Comment:
took me a while to realize this, but django actually automatically instantiates `self.client` as `Client()`